frontend/public/components/edit-yaml.jsx

```diff
@@ -460,11 +478,17 @@ export const EditYAML = connect(stateToProps)(
     }
 
     registerYAMLHover(languageID, monaco, m2p, p2m, createDocument, yamlService) {
+      const that = this;
       monaco.languages.registerHoverProvider(languageID, {
         provideHover(model, position) {
           const document = createDocument(model);
           return yamlService.doHover(document, m2p.asPosition(position.lineNumber, position.column)).then((hover) => {
             return p2m.asHover(hover);
+          }).then(e => {
+            that.setState({
+              isHovering: true
```

Comment:
Does this need to be tracked in component state? I'd expect that we could simply call `editorDidHover` here and remove the `isHovering` state value.